src/Edge.java

```diff
@@ -0,0 +1,19 @@
+public class Edge {
```

Comment:
This whole class now looks redundant for the trie. We'll need it for the dawg, I guess, but you can nuke it from here.